src/main/java/com/stripe/Stripe.java

```diff
@@ -34,6 +34,18 @@ public abstract class Stripe {
    */
   public static volatile String stripeVersion = API_VERSION;
 
+  /** Add a specified beta to the global Stripe API Version. Only call this method once per beta. */
+  public static void addBetaVersion(String betaName, String betaVersion) {
+    if (stripeVersion.indexOf("; " + betaName) >= 0) {
```

Comment:
```suggestion
    if (stripeVersion.indexOf("; " + betaName) >= 0 + "=") {
```
worrying about the case where one beta name is the prefix of another

Reply:
Good point!